Preserve GNU libtextstyle licence and attribution on the generated header.
Origin: GNU gettext 1.0 libtextstyle/gnulib-local/lib/libcroco.
EmberBSD integration: AI-assisted; not submitted upstream.

--- src/libcroco-config.h.in.orig
+++ src/libcroco-config.h.in
@@ -1,3 +1,23 @@
+/* libcroco - Library for parsing and applying CSS
+ * Copyright (C) 2006-2019 Free Software Foundation, Inc.
+ *
+ * This file is not part of the GNU gettext program, but is used with
+ * GNU gettext.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
 #ifndef LIBCROCO_VERSION_NUMBER
 #define LIBCROCO_VERSION_NUMBER @LIBCROCO_VERSION_NUMBER@
 #endif
